Realm/Realm/Helpers/Argument.cs

```diff
@@ -95,7 +95,7 @@ public static T ValidateNotNull<T>(T value, string paramName)
 
         public static void AssertDebug(string message)
         {
-            Logger.LogDefault(LogLevel.Error, $"{message} {OpenIssueText}");
+            RealmLogger.LogDefault(LogLevel.Error, $"{message} {OpenIssueText}");
```

Comment:
We should settle on either `RealmLogger.LogDefault` or `RealmLogger.Default.Log` - having both is confusing and opens us up for slight inconsistencies.

Reply:
I agree, I can incorporate that into this PR 👍  Since the `LogDefault` is internal, I'll update the usage to the public-facing `Default.Log()`.